server/db/controllers/articleController.js

```diff
@@ -34,20 +35,22 @@ module.exports = {
           cb(article);
         })
         .catch((err) => {
-          cb('Error saving article to database:', err);
+          throw ('Error saving article to database:', err);
```

Comment:
did you mean to throw an error here? (is it being caught and handled elsewhere?)